app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlController.kt

```diff
@@ -335,15 +339,20 @@ class DefaultSessionControlController(
         val directions = HomeFragmentDirections.actionHomeFragmentToBrowserFragment(null)
         val extras =
             FragmentNavigator.Extras.Builder()
-                .addSharedElement(tabView,
+                .addSharedElement(
+                    tabView,
                     "$TAB_ITEM_TRANSITION_NAME$sessionId"
                 )
                 .build()
         navController.nav(R.id.homeFragment, directions, extras)
     }
 
     override fun handleSelectTopSite(url: String) {
-        metrics.track(Event.TopSiteOpenInNewTab)
+        if (url == SupportUtils.POCKET_TRENDING_URL) {
+            metrics.track(Event.PocketTopSiteClicked)
```

Comment:
Do we also want to track TopSiteOpen here as well?

Reply:
I am in the else branch 😅